.nais/naiserator-dev.yaml

```diff
@@ -88,6 +91,8 @@ spec:
   kafka:
     pool: nav-dev
   leaderElection: true
+  envFrom:
+    - configmap: loginservice-idporten
```

Comment:
Kanskje bare skrive om til tokenx med en gang

Reply:
Helt enig. Jeg har laget et kort på trello for å gå vekk fra loginservice i esyfo-info-frontend, så det du sier kan bli gjort samtidig.